### DISCUSSION TITLE
Welcome to Kontraption Discussions!

### DISCUSSION THREAD

**Discussion by illucc** [illucc/Kontraption#23] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by illucc** [comment]
this thingy is here for people that dont use discord

**Reply by TuffetSpider** [reply]
bonjor it is i tuffet

**Reply by AdamProGamingTR** [reply]
reallllllll

**Reply by illucc** [reply]
~~github discord~~

**Reply by AdamProGamingTR** [reply]
skullll

**Reply by TuffetSpider** [reply]
github: the (o̶l̶d̶) new discord

**Reply by jhammerberg** [reply]
Just discovered this mod, it's really cool! I've been looking for something that's a more modern version of Warpdrive and this looks really promising. Are there any plans to add space travel and/or multiple planets in the future?

**Reply by illucc** [reply]
> Just discovered this mod, it's really cool! I've been looking for something that's a more modern version of Warpdrive and this looks really promising. Are there any plans to add space travel and/or multiple planets in the future?

Kontraption itself wont add space but if some space mod releases ill try to make compat with it 👍

**Reply by AdamProGamingTR** [reply]
@illucc when is my translation getting added smh

**Reply by illucc** [reply]
@AdamProGamingTR when im not combusting exploding and dying irl

**Reply by AdamProGamingTR** [reply]
understandable skul
